packages/interface/src/components/fullscreen-mode/index.js

```diff
@@ -22,6 +22,10 @@ export class FullscreenMode extends Component {
 		if ( this.isSticky ) {
 			document.body.classList.add( 'sticky-menu' );
 		}
+
+		if ( this.props.isActive ) {
+			document.body.classList.remove( 'is-fullscreen-mode' );
+		}
```

Comment:
I think rewriting that component as hook is probably the best path forward. At the time we introduce this one, these were not available.

Reply:
Agreed. With `useEffect`, the code will be much shorter. I'll do it in the next PR. In this one, I wanted to focus on the actual bugfix.